ui/src/components/chat/sidebar.rs

```diff
@@ -380,6 +414,17 @@ pub fn Sidebar(cx: Scope<Props>) -> Element {
                                         state.write().mutate(Action::RemoveFromSidebar(chat.id));
                                     }
                                 },
+                                hr{ }
+                                ContextItem {
+                                    icon: Icon::Trash,
+                                    danger: true,
+                                    text: if is_group_conv && is_admin {get_local_text("uplink.delete-group-chat")} 
+                                    else if is_group_conv && !is_admin  {get_local_text("uplink.leave-group")} 
+                                    else {get_local_text("uplink.delete-conversation")},
+                                    onpress: move |_| {
+                                        ch.send(MessagesCommand::DeleteConversation { conv_id: chat.id });
```

Comment:
we probably want to disable this `ContextItem` while the `MessagesCommand` is being processed.